src/components/MultiHopTrade/components/RateGasRow.tsx

```diff
@@ -44,11 +46,38 @@ export const RateGasRow: FC<RateGasRowProps> = memo(
     swapperName,
     swapSource,
     networkFeeFiatUserCurrency,
+    deltaPercentage,
     onClick,
   }) => {
     const translate = useTranslate()
     const { isOpen, onToggle } = useDisclosure()
 
+    const deltaPercentagePercentage = useMemo(() => {
+      if (!deltaPercentage) return null
+
+      if (deltaPercentage.isGreaterThan(0.01) || deltaPercentage.isLessThan(-0.01)) {
```

Comment:
`gt(0)` / `lt(0)` / `isZero()` and remove this branch?

Reply:
Removed in [e905124](https://github.com/shapeshift/web/pull/8970/commits/e9051249766cda9ac47762f1ce453ba3f1e35c9f)